src/monocle_apptrace/metamodel/entities/README.md

```diff
@@ -0,0 +1,43 @@
+# Monocle Entities
+The entity type defines the type of GenAI component that Monocle understand. The monocle instrumentation can extract the relevenat information for this entity. There are a fixed set of [entity types](./entity_types.py) that are defined by Monocle out of the box, eg workflow, model etc. As the GenAI landscape evolves, the Monocle community will introduce a new entity type if the current entities won't represent a new technology component.
+
+## Entity Types
```

Comment:
can all the entity types be small case

Reply:
I would like to keep the vendor names starting with upper case. Do you have a strong preference all lowercase or if that's the convention generally followed, then we can switch to lower case.

Reply:
'AWS` - capital case
'Azure` - pascal case
`NVIDIA` - capital case
`OpenAI`- pascal case
`HiggingFace` - pascal case

as type is in our control, we can keep it consistent.

Reply:
True .. I was trying to keep it consistent with the vendor name/brand, but I see your point. I guess it's better to be consistent here. Will change it all to all lowercase and camel case.